docs/applications_general.py

```diff
@@ -0,0 +1,211 @@
+#In this example, we use a Python data flow scripting interface to connect
+#PyEmma, a simulator adapter, and some additional Python code to iteratively
+#run and analyze groups of simulations until convergence.
+#
+#The main script body defines and executes the subgraph.
+#
+#One supporting module (`analyze.py`) wraps PyEmma.
+#
+#supporting modules (`gromacs_wrapper.py` and `lammps_wrapper.py`) 
+#wraps command line MD commands with a SCALE-MS wrapper layer.  The results should be in the same format.
+
+from gromacs_wrapper import collect_configurations, structure_to_pdb, make_input, simulate, modify_input
+
+# configuration_list (an array of filenames) and md_inputs (a file
+# name, or list of file names if multiple inputs are needed to
+# intitialize the simulation.
+#
+# In theory, this could be abstracted further as some sort of input bundle.
+# that contains all information needed to star the simulation.
+# 
+# So if one has N simulations, you would start with a list of N input objects.
+# Each object could be tagged as a package of a supported type (GROMACS, LAMMPS, NAMD),
+# and wrapper code for each supported type knows how to unwrap it.
+#
+# for GROMACS, it would be a .tpr file (or internal represntation of
+# the .tpr file) For LAMMPS, it's the array of input files, since
```

Comment:
I think it would be either a gmxapi.SimulationInput object or a lammps.SimulationInput object, each of which can support some amount of a generic scalems.SimulationInput interface.

Reply:
OK, we will need to talk more about this.